openeo/extra/job_management.py

```diff
@@ -192,24 +192,17 @@ def _normalize_df(self, df: pd.DataFrame) -> pd.DataFrame:
             ("duration", None),
             ("backend_name", None),
         ]
-        new_columns = {
-            col: val for (col, val) in required_with_default if col not in df.columns
-        }
+        new_columns = {col: val for (col, val) in required_with_default if col not in df.columns}
         df = df.assign(**new_columns)
-        # Workaround for loading of geopandas "geometry" column.
-        if "geometry" in df.columns and df["geometry"].dtype.name != "geometry":
-            df["geometry"] = df["geometry"].apply(shapely.wkt.loads)
```

Comment:
This workaround seems to be removed by this PR. Is that intended?
I think it's still necessary for at least the CSV use case?

Reply:
It's indeed intended, I want to leave all data in the geometry column as a string. Otherwise we run into trouble when writing the pandas DF to Parquet. Is there a reason why it might be necessary for CSV? Leaving it as a string also seems to work fine.

Reply:
But is removing this going to break some use cases where users expect a real geometry in their `start_job` callable?

Reply:
Hmm yes probably. So I can add it to the read method of CsvJobDatabase, but not to the ParquetJobDatabase, or what do you propose?

Reply:
indeed, I would reinstate it at least for the CSV use case.

With parquet, the geometry is maybe loaded correctly automatically?

Reply:
Both for CSV and Parquet, pandas will parse the geometry as string. However, for writing, it's possible to write a real geometry to CSV, but not to parquet (you'd have to use geopandas instead of pandas)